### PR TITLE
Feat/sort filter

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. sort by latest to oldest and vice versa
2. filter by labels
3. other responsive changes

### CODE REVIEW

File: src/context/sort-filter-context.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import {
+  createContext,
+  useContext,
+  useReducer, 
+  useState
+} from "react";
+import { useEffect } from "react";
```
Comment (ivinayakg) [inline src/context/sort-filter-context.js:7]: importing from the package

File: src/reducer/sort-filter-reducer.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const sortFilterInitialState = {
+  sortBy: '',
+  filterBylabels: []
+}
+const sortFilterReducer = (state, action) => {
+  console.log(action)
```
Comment (ivinayakg) [inline src/reducer/sort-filter-reducer.js:6]: remove console

**Review by ivinayakg — APPROVED** [review]
Well written code, Great Job 👍

File: src/component/search-bar/search-bar.css [hunk 3]
```diff
@@ -19,3 +19,33 @@
 .search-bar-input:focus {
   border-bottom: 1px solid var(--dark-secondary-text-color);
 }
+.overlay{
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
```
Comment (darshandamre) [inline src/component/search-bar/search-bar.css:27]: you can replace these four lines with
```css
inset: 0
```

**Review by darshandamre — APPROVED** [review]
liked the usage of react portals

**Pull request merged**